src/renderers/shared/fiber/ReactFiberReconciler.js

```diff
@@ -83,6 +83,7 @@ export type HostConfig<T, P, I, TI, PI, C, CX, PL> = {
 
   shouldSetTextContent(props : P) : boolean,
   resetTextContent(instance : I) : void,
+  areChildrenOffscreen(props : P) : boolean,
```

Comment:
`shouldChildrenBeOffscreen` doesn't say anything about priorities and just refers to the offscreenness which could imply invisibility or many other things.

It's also unclear to me what will happen to the Offscreen priority level when we have expiration times. Maybe it's just a very long expiration but not actually anything magically later than anything else. Avoids the problem of starving the offscreen content and running out of memory adding things to the queue.

I like `shouldDeprioritizeChildren` since it refers to priorities. Maybe `shouldDeprioritizeSubtree` because it is not just direct children but also grand children?